content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -29,13 +29,24 @@ For clarity, this guide defines the following terms:
 {{< link text="services" url="/docs/concepts/services-networking/service/" >}} within the cluster.
 Traffic routing is controlled by rules defined on the Ingress resource.
 
-```none
-    internet
-        |
-   [ Ingress ]
-   --|-----|--
-   [ Services ]
-```
+Here is a simple example where an Ingress sends all its traffic to one Service:
+{{< mermaid >}}
+graph LR;
+  client([client])-. Ingress-managed load balancer .->ingress[Ingress];
```

Comment:
Hello @sftim . Some feedback:
- The mermaid.js graphics look much richer on the page.
- The first diagram fits into the page, but the other diagrams in this change overflow the container to the right.
Is there a stylesheet or configuration for the site's mermaid diagrams? Before more diagrams are added,
perhaps a stylesheet can be created.
- When I see this string, `Ingress-managed load balancer` outside of the cluster box, my first
thought is where is the load balancer running.
- The cluster is represented by a box. Could you use
a cluster icon to represent a cluster?
- I would like to sort out the page reload for pages that include mermaid.js. I doubt that other sites show the mermaid.js raw markdown when a page reloads.

Reply:
> The first diagram fits into the page, but the other diagrams in this change overflow the container to the right.

Can you provide a screenshot or some details of the setup you have @kbhawkey ? I don't see that here.

>  Is there a stylesheet or configuration for the site's mermaid diagrams?
Not right now - that could be an enhancement issue.

> The cluster is represented by a box. Could you use a cluster icon to represent a cluster?

MermaidJS does not provide support for using the Kubernetes icon set.

> I would like to sort out the page reload for pages that include mermaid.js. I doubt that other sites show the mermaid.js raw markdown when a page reloads.

We could render MermaidJS during static site build. That's a chunk of work to get right though!
We could also render the diagrams locally to SVG, commit those, and skip using MermaidJS apart from for local development.

Would that work?

Reply:
I am in favor of the diagrams fitting or resizing with the page.
Creating SVGs locally using MermaidJS makes sense unless there is a reason to generate the images when the page loads.
Here are a couple of snippets of the ingress page (laptop screen).


![ingress_diag_2](https://user-images.githubusercontent.com/1696019/91092720-712ad680-e626-11ea-9541-2f9e7e39c1bf.PNG)

![chrome_ingress_mermaid](https://user-images.githubusercontent.com/1696019/91093029-d5e63100-e626-11ea-8fa3-c374d3c4ff51.PNG)

Reply:
@sftim I think `<br>` tag can be used in the text. This should shrink the width of the labels. So "Ingress-managed load balancer" and "Ingress, 178.91.123.132" can be "Ingress-managed<br>load balancer" and "Ingress<br>178.91.123.132".